### PR TITLE
fix: add org id to planter endpoints to so orgs can edit planters

### PR DESCRIPTION
**Pull request by gwynndp**

### CODE REVIEW

File: src/controllers/planter.controller.ts [hunk 1]
```diff
@@ -74,14 +74,15 @@ export class PlanterController {
     return await this.planterRepository.findById(id);
   }
 
-  @get('/planter/{id}/selfies', {
+  @get('/organization/{organizationId}/planter/{id}/selfies', {
```
Comment (nmcharlton) [inline src/controllers/planter.controller.ts]: This endpoint is already implemented in `planterOrganization.controller.ts`.

File: src/controllers/planterRegistration.controller.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export class PlanterRegistrationController {
     public planterRepository: PlanterRegistrationRepository,
   ) {}
 
-  @get('/planter-registration', {
+  @get('/organization/{organizationId}/planter-registration', {
```
Comment (nmcharlton) [inline src/controllers/planterRegistration.controller.ts]: I think we'll need to keep the original endpoint for non-org requests.
The `trees` and `planter` org endpoints are implemented in separate controllers: `treesOrganization.controller.ts` and `planterOrganization.controller.ts` respectively.

File: src/controllers/planter.controller.ts (hunk 1 above)
Comment (gwynndp) [reply src/controllers/planter.controller.ts]: The existing endpoint wasn't responding to the "/selfies" request.  I can add the /selfies endpoint with the org id to the planterOrganization.controller.ts and keep the non-org endpoint on planter.controller.

File: src/controllers/planterRegistration.controller.ts (hunk 2 above)
Comment (gwynndp) [reply src/controllers/planterRegistration.controller.ts]: Agreed. I can add the /planter-registration endpoint with the org id to the planterOrganization.controller.ts and keep the non-org endpoint on planter.controller.

**Review by nmcharlton — APPROVED** [review]
Great work, as always 🚀

**Pull request merged**